### PR TITLE
add getting Storage Quantity to ResourceList

### PR DESCRIPTION
**Pull request by yuyulei**
*What type of PR is this?**
> /kind feature

**What this PR does / why we need it**:
easily get storage quantity of pv or pvc

```release-note
NONE
```

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
/ok-to-test

change makes sense to me, aligns with other standard resource helpers.

/approve
/lgtm

**Comment by yuyulei** [comment]
@derekchiang

**Comment by yuyulei** [comment]
test passed PTAL @derekwaynecarr

**Comment by yuyulei** [comment]
@derekwaynecarr

**Comment by yuyulei** [comment]
PTAL @nikhiljindal

**Comment by yuyulei** [comment]
ptal @liggitt

**Comment by yuyulei** [comment]
PTAL @nikhiljindal

File: staging/src/k8s.io/api/core/v1/resource.go [hunk 1]
```diff
@@ -41,6 +41,14 @@ func (self *ResourceList) Memory() *resource.Quantity {
 	return &resource.Quantity{Format: resource.BinarySI}
 }
 
+// Returns the Storage request of VolumeClaimTemplates, or Storage Capacity of PersistentVolume.
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/resource.go]: remove references to specific API types from the godoc

```suggestion
// Returns the Storage limit if specified.
```

Comment (yuyulei) [reply staging/src/k8s.io/api/core/v1/resource.go]: ok, ptal.

Comment (yuyulei) [reply staging/src/k8s.io/api/core/v1/resource.go]: ptal.

**Comment by yuyulei** [comment]
@liggitt ptal

**Comment by yuyulei** [comment]
@liggitt @derekwaynecarr

**Comment by liggitt** [comment]
address the verify failure on godoc, then lgtm

**Comment by yuyulei** [comment]
@liggitt verify-test has passed

**Pull request merged**